### PR TITLE
Update CHANGELOG for 1.5.0-beta.0

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,168 @@
 # Release Notes
 
+## Version 1.5.0-beta.0 - 2019-10-15
+
+* Replace registry-creds addon ReplicationController with Deployment [#5586](https://github.com/kubernetes/minikube/pull/5586)
+* Performance and security enhancement for ingress-dns addon [#5614](https://github.com/kubernetes/minikube/pull/5614)
+* Add addons flag to 'minikube start' in order to enable specified addons [#5543](https://github.com/kubernetes/minikube/pull/5543)
+* Disable go modules for containerd, for newer go [#5608](https://github.com/kubernetes/minikube/pull/5608)
+* Upgrade Buildroot to 2019.02 and VirtualBox to 5.2 [#5609](https://github.com/kubernetes/minikube/pull/5609)
+* gsutil: Remove trailing slashes, rsync instead of cp [#5620](https://github.com/kubernetes/minikube/pull/5620)
```
Comment (tstromberg) [inline CHANGELOG.md]: The changelog should contain user visible changes only, and specifically exclude any changes which are not shipped as part of the binary or ISO. Specifically, the changelog should omit:

- CI changes
- Documentation-only changes
- Test-only changes

It doesn't have to be exact, but filter as best as you can for them. The fewer entries the better, IMHO. 

(We should open a bug to automatically filtering out changes by file extension in the future)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,168 @@
...
+* Improve out test coverage [#5445](https://github.com/kubernetes/minikube/pull/5445)
+* Fix node InternalIP not matching host-only address [#5427](https://github.com/kubernetes/minikube/pull/5427)
+* Fix crash when deleting the cluster but it doesn't exist [#4980](https://github.com/kubernetes/minikube/pull/4980)
+* Improve storageclass test coverage [#5356](https://github.com/kubernetes/minikube/pull/5356)
+* Remove all functions from constants.go [#5409](https://github.com/kubernetes/minikube/pull/5409)
+* improve test coverage for sshutil package [#5283](https://github.com/kubernetes/minikube/pull/5283)
+* improve test coverage in service pkg [#5258](https://github.com/kubernetes/minikube/pull/5258)
+
+Huge thank you for this release towards our contributors: 
+- Abdulla Bin Mustaqeem
```
Comment (tstromberg) [inline CHANGELOG.md]: I'm suspicious of this list: we have 54 PR's and 116 contributors?

Comment (sharifelgamal) [reply CHANGELOG.md]: I did delete a pretty large number of PRs, but I can double check what the script is doing and see if anything's wrong.

File: CHANGELOG.md (hunk 1 above)
Comment (sharifelgamal) [reply CHANGELOG.md]: Done, remaining PRs should be user facing. Let me know if I missed any I should delete.

File: CHANGELOG.md (hunk 2 above)
Comment (tstromberg) [reply CHANGELOG.md]: I think our script may be wrong. This should output the approximate list of contributors:

`git log refs/tags/v1.4.0.. --format=%aN --reverse | sort | uniq -w10 | grep -v "bot\$"`

(26 people, minus some obvious dupes)

Comment (sharifelgamal) [reply CHANGELOG.md]: that does seem a lot more reasonable. fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,45 +5,27 @@
 * Replace registry-creds addon ReplicationController with Deployment [#5586](https://github.com/kubernetes/minikube/pull/5586)
 * Performance and security enhancement for ingress-dns addon [#5614](https://github.com/kubernetes/minikube/pull/5614)
 * Add addons flag to 'minikube start' in order to enable specified addons [#5543](https://github.com/kubernetes/minikube/pull/5543)
-* Disable go modules for containerd, for newer go [#5608](https://github.com/kubernetes/minikube/pull/5608)
 * Upgrade Buildroot to 2019.02 and VirtualBox to 5.2 [#5609](https://github.com/kubernetes/minikube/pull/5609)
-* gsutil: Remove trailing slashes, rsync instead of cp [#5620](https://github.com/kubernetes/minikube/pull/5620)
 * Update default Kubernetes version to 1.16.1 [#5593](https://github.com/kubernetes/minikube/pull/5593)
 * Initial translations for fr, es, de, ja, and zh-CN [#5466](https://github.com/kubernetes/minikube/pull/5466)
 * Add libmachine debug logs back [#5574](https://github.com/kubernetes/minikube/pull/5574)
 * Add TCP/UDP ingress tutorial [#5517](https://github.com/kubernetes/minikube/pull/5517)
```
Comment (tstromberg) [inline CHANGELOG.md]: Can be removed

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,45 +5,27 @@
...
 * Bump golangci-lint to v1.20.0 [#5572](https://github.com/kubernetes/minikube/pull/5572)
```
Comment (tstromberg) [inline CHANGELOG.md]: Can be removed

**Pull request merged**